File: Foodie/ClientApp/src/app/models/meal.ts
import { Ingriedient } from "./ingiedient";

export interface Meal{
    id: number;
    name: string;
    date: Date;
    ingriedients: Ingriedient[];
    portions: number;
    panelColor: string;
    tableColor: string;
    deleteForbid:boolean;
}

File: Foodie/Models/Meal.cs
using System;
using System.Collections.Generic;

namespace Foodie.Models
{
    public class Meal
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public DateTime Date { get; set; }
        public List<Ingriedient> Ingriedients { get; set; }
        public ApplicationUser User { get; set; }
    }
}